src/components/pages/login/index.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import LoginForm from './loginForm';
-import logo from '../../../../images/landingPage/logo.png';
+import newBquest from '../../../../images/loginPage/white-bquest.svg.inline'
+import newBquestLogo from '../../../../images/loginPage/white-bquest-logo.svg.inline'
 
 class Login extends React.Component {
 
     render() {
         return(
             <div className="container-login">
                 <div className="logo">
-                    <img src={logo} id="logo" alt="logo" />
+
+					{/*<img className="logo__logo" src={newBquestLogo} alt="newBquestLogo"/>*/}
+					{/*<img className="logo__text" src={newBquest} alt="newBquest"/>*/}
+					<div className="logo__logo" dangerouslySetInnerHTML={{__html: newBquestLogo}} />
+					<div className="logo__text" dangerouslySetInnerHTML={{__html: newBquest}} />
                 </div>
                 <div className="container-loginform">
-                    Create an account <Link className="register-color" to="/register">here</Link> if you do not have one
+					<h1 className="loginform__title">Login</h1>
```

Comment:
just for consistency, please keep classNames with regular en-dash, rather than underscores

Reply:
Okay. (I originally want to probably comply with BEM model though)